cg/models/workflow/validators.py

```diff
@@ -0,0 +1,2 @@
+def get_first_word(value: str) -> str:
```

Comment:
```suggestion
def get_first_word(sentance: str) -> str:
```
Seems not specific to Mutant - should we create a general Pydantic validators module? aving the same discussion here with @fevac @islean https://github.com/Clinical-Genomics/cg/pull/2367#discussion_r1312628865)

Reply:
Yeah, I think it is worth having a discussion about it, although it might be worth structuring a bit more. Either by using sub directories or keeping quite strict in ordering of the validators in the file (to the extent possible).

I would also add that this could probably be an AfterValidator. It would probably make more sense to get a parsing error for wrong type than getting an error stating that "split" does not work for the input.

Reply:
Sure, that could make sense. But we should only move stuff that is used elsewhere. In practice, I think most of the "general" validators will only get used in a single model anyway and then it makes more sense to keep it close to the model

Reply:
Out of scope for this PR, lets create it once we stumble across a validator that is used elsewhere